src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/TypecastParenPadCheck.java

```diff
@@ -41,13 +41,37 @@
  * &lt;module name=&quot;TypecastParenPad&quot;/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * float f1 = 3.14f;
+ *
+ * int n = ( int ) f1; // violation, space after left parenthesis and before right parenthesis
```

Comment:
please add 2 more examples with one whitespace after left paren and with one after right paren